roles/aap_ocp_install/README.md

```diff
@@ -89,14 +115,41 @@ If the variable is omitted the corresponding component will not be installed (e.
 | link_text                          |          | EDA Controller (<INSTANCE_NAME>) | Text used for creating the OCP application link |
 | eda_manifest_overrides             |          | None                             | YAML Manifest to override the generated `EDA` resource  |
 | consolelink_manifest_overrides     |          | None                             | YAML Manifest to override the generated `ConsoleLink` resource    |
+| install                            | *        | false                            | Whether or not to install the EDA platform component in AAP 2.5 or later    |
+
+\* These settings are only used for installing AAP 2.5 or later.
+
+> ℹ️ **NOTE**
+>
+> The namespace, instance_name and link_text values will be ignored when using the platform installation method.
+
+### aap_ocp_install_platform keys
+
+| Key Name      | Required | Default Value                    | Description                                     |
+|---------------|:--------:|----------------------------------|-------------------------------------------------|
+| instance_name | Yes      | None                             | Name of the AAP Platform instance to create     |
```

Comment:
Can you document the `namespace` and `link_text` options here. You can just copy/paste from one of the other sections.